homeassistant/components/melissa/climate.py

```diff
@@ -136,6 +139,17 @@ def max_temp(self):
         """Return the maximum supported temperature for the thermostat."""
         return 30
 
+    @property
+    def device_info(self) -> DeviceInfo:
```

Comment:
I think this should be using self._attr_device_info, and it should be done in a follow-up PR to avoid distracting from the actual config-flow.